docs/_pages/typecheck.md

```diff
@@ -347,7 +347,7 @@ And using `assert` will work with the above type guards:
 ```lua
 local stringOrNumber: string | number = "foo"
 
-assert(type(x) == "string")
+assert(type(stringOrNumber) == "string", "stringOrNumber is a string")
```

Comment:
Thanks - `x` was a mistake. Any specific reason this needs a second argument?

Reply:
Whenever I try to use `assert()` without a second arg, I receive this error:

![image](https://user-images.githubusercontent.com/13839150/128217248-070277b7-7c17-4c81-a71b-30ed42e4bf4a.png)

Reply:
Is this Selene? In any case this is wrong, `assert`'s second parameter is optional.

Reply:
Ahhhhhh, useful knowledge, thank you :-)